src/azure-cli/azure/cli/command_modules/vm/_validators.py

```diff
@@ -1635,17 +1640,21 @@ def _validate_vmss_create_load_balancer_or_app_gateway(cmd, namespace):
                 raise CLIError('usage error:{}'.format(err))
 
 
-def get_network_client(cli_ctx):
-    from azure.cli.core.profiles import ResourceType
-    from azure.cli.core.commands.client_factory import get_mgmt_service_client
-    return get_mgmt_service_client(cli_ctx, ResourceType.MGMT_NETWORK)
+def get_network_client(cli_ctx, balancer_type):
+    if balancer_type == 'application_gateways':
+        return import_aaz_by_profile(cli_ctx.cloud.profile, "network.application_gateway")
+    if balancer_type == 'load_balancers':
+        return import_aaz_by_profile(cli_ctx.cloud.profile, "network.lb")
```

Comment:
Mark

Reply:
I will remove this function.

Reply:
It's the old implementation by @PARADISSEEKR

Reply:
OK~